lib/apia/open_api/rack.rb

```diff
@@ -46,7 +50,11 @@ def call(env)
           return @app.call(env)
         end
 
-        specification = Specification.new(api_class, base_url, @options[:name], info, external_docs)
+        specification = Specification.new(api_class, base_url, @options[:name], {
+          info: info,
+          external_docs: external_docs,
+          security_schemes: security_schemes
+      })
```

Comment:
nit: the alignment is off here, the `})` should line up with the start of `specification`